src/main/python/deep_qa/data/instances/question_passage_instance.py

```diff
@@ -99,7 +99,8 @@ def pad(self, max_lengths: Dict[str, int]):
         max_lengths_tmp['word_sequence_length'] = max_lengths_tmp['num_question_words']
         self.question_indices = self.pad_word_sequence(self.question_indices, max_lengths_tmp)
         max_lengths_tmp['word_sequence_length'] = max_lengths_tmp['num_passage_words']
-        self.passage_indices = self.pad_word_sequence(self.passage_indices, max_lengths_tmp)
+        self.passage_indices = self.pad_word_sequence(self.passage_indices, max_lengths_tmp,
```

Comment:
im not sure how i feel about this, since it affects models like the GAReader / ASReader. is there anyway you can change this just for BiDAF?